dev-requirements.txt

```diff
@@ -1,22 +1,18 @@
 mock==2.0.0
-coverage==4.5.2
-twine==1.11.0
+coverage~=4.5
+tox==2.9.1
```

Comment:
This is not really useful since it's tox who installs those requirements, so we don't really check that this version works. And in this case I don't think tox 2.9.1 works as it does not support PEP 517. (But we already had the issue before the PR.)